examples/ingest/elasticsearch/ingest.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Runs a docker container to create an elasticsearch cluster,
+# fills the ES cluster with data,
+# processes all the files in the 'movies' index in the cluster using the `unstructured` library.
+
+# Structured outputs are stored in elasticsearch-ingest-output
+
+# TODO: provide inputs / outputs to test automatically
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+# Create the elasticsearch cluster and get the container id
+output=$(docker run -d --rm -p 9200:9200 -p 9300:9300 -e "xpack.security.enabled=false" -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:8.7.0)
+container_id=$(echo "$output" | cut -c 1-12)
+
+
+url="http://localhost:9200/_cluster/health"
+status_code=0
+retry_count=0
+max_retries=6
+
+# Check the cluster status repeatedly until it becomes live or maximum retries are reached
+while [ "$status_code" -ne 200 ] && [ "$retry_count" -lt "$max_retries" ]; do
+
+  # Send a GET request to the cluster health API
+  response=$(curl -s -o /dev/null -w "%{http_code}" "$url")
+  status_code="$response"
+
+  # Process the files only when the ES cluster is live
+  if [ "$status_code" -eq 200 ]; then
+    echo "Cluster is live."
+
+    python examples/ingest/elasticsearch/elasticsearch_cluster.py
+
+    PYTHONPATH=. ./unstructured/ingest/main.py \
+            --elasticsearch-url http://localhost:9200 \
+            --elasticsearch-index-name movies \
+            --jq-query '{ethnicity, director}' \
```

Comment:
how about we add plot in here? since that's the beefiest field?

Reply:
Sounds good 👍